Makefile.in

```diff
@@ -228,6 +236,10 @@ config_files: ocaml-stage1-config.status
 	/bin/echo -n "( `grep \"^SHAREDLIB_CFLAGS=\" ocaml/Makefile.config \
 		| sed 's/^SHAREDLIB_CFLAGS=//'` )" > sharedlib_cflags.sexp
 
+.PHONY: ocaml_compiler_internal_param
```

Comment:
Does this target name deliberately lack the final "s"?  I'd prefer it to have the "s" but maybe something will go wrong.